frontend/src/container/MetricsApplication/Tabs/Overview/TopOperationMetrics.tsx

```diff
@@ -58,18 +56,15 @@ function TopOperationMetrics(): JSX.Element {
 
 	const updatedQuery = useStepInterval(keyOperationWidget.query);
 
-	const isEmptyWidget = useMemo(
-		() => keyOperationWidget.id === 'empty' || isEmpty(keyOperationWidget),
-		[keyOperationWidget],
-	);
+	const isEmptyWidget = keyOperationWidget.id === PANEL_TYPES.EMPTY_WIDGET;
 
 	const { data, isLoading } = useGetQueryRange(
 		{
 			selectedTime: keyOperationWidget?.timePreferance,
 			graphType: keyOperationWidget?.panelTypes,
 			query: updatedQuery,
 			globalSelectedInterval,
-			variables: getDashboardVariables(),
+			variables: {},
```

Comment:
Empty Object for variables ?

Reply:
yes, there is no dashboard under the metrics application... that's the reason empty object